### ISSUE TITLE
Could not load stories - require.context is not a function

### ISSUE THREAD

**Issue by tleunen** [storybooks/storyshots#66]
After updating the libraries and creating the storyshots test file, I'm having this error.
```
Could not load stories from /Users/xxx/dev/yyy/.storybook/config.js. Check 'configPath' option
```

The thing is, the file exists. So after hacking the code and displaying the true error, I can see this one:
```
TypeError: require.context is not a function
```
And indeed, my config file has a require.context function to automatically include all the stories in my source directory for the storybook.

It seems storyshots doesn't do anything with the configuration, would it be possible to change it for something like `fs.existsSync()`?

**Comment by arunoda** [comment]
@roonyh is there anything we can do about this?
I hope we can do something here.

And it's better to display the actual error message as well.

**Comment by kitze** [comment]
@arunoda I'm hitting the same problem.

Here's my storybook config:
```
import wrapWithMui from '../src/wrappers/wrap-mui';
import wrapWithStyles from '../src/wrappers/wrap-styles';
import {addDecorator, configure} from '@kadira/storybook';

var reqComponents = require.context('components', true, /.stories.js$/)
var reqViews = require.context('views', true, /.stories.js$/)

addDecorator(story => wrapWithStyles(story()));
addDecorator(story => wrapWithMui(story()));

const loadStories = () => {
  reqComponents.keys().forEach(filename => reqComponents(filename));
  reqViews.keys().forEach(filename => reqViews(filename));
}

configure(loadStories, module);
```

And here's [the error](http://jmp.sh/DvHTm0H) I'm getting.

**Comment by arunoda** [comment]
Released #67 with `v3.1.0`
I hope it'll fix this issue.

**Comment by tleunen** [comment]
Confirmed, fixed :)
